tests/test.js

```diff
@@ -114,10 +116,13 @@ function test_options(Module) {
  * @param {import("../types").Highs} Module
  */
 function test_invalid_model(Module) {
-  assert.throws(
-    (_) => Module.solve("blah blah not a good file"),
-    /Unable to read LP model/
-  );
+  const sol = Module.solve("blah blah not a good file");
+  assert.deepStrictEqual(sol, {
```

Comment:
That is a regression

Reply:
Right, I'd probably also expect that particular input to fail. How about we raise that in `ergo-code/HiGHS` though, then come up with another example of something that still fails here?

Reply:
Is there no way to detect that the input was incorrect here?

Reply:
Not really; as the test suggests we do indeed get a meaningful output out of it.

Reply:
I added another input that HiGHS won't except in another test, and created an issue for the original one here: https://github.com/ERGO-Code/HiGHS/issues/1451